packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -2963,6 +2963,133 @@ void main() {
     expect(tester.getTopRight(find.text('hint')).dx, 760.0);
   });
 
+  testWidgets('FloatingLabelAlignment.toString()', (WidgetTester tester) async {
+    expect(FloatingLabelAlignment.start.toString(), 'FloatingLabelAlignment(x: -1.0)');
+    expect(FloatingLabelAlignment.center.toString(), 'FloatingLabelAlignment(x: 0.0)');
+  });
+
+  group('floatingLabelAlignment', () {
+    Widget buildID({required TextDirection td, required bool icon, required FloatingLabelAlignment fla}) {
```

Comment:
Nit: Consider replacing these abbreviations with the full words, per the style guide. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations